bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -512,6 +512,143 @@ type BootstrapTokenDiscovery struct {
 type FileDiscovery struct {
 	// KubeConfigPath is used to specify the actual file path or URL to the kubeconfig file from which to load cluster information
 	KubeConfigPath string `json:"kubeConfigPath"`
+
+	// KubeConfig is used (optionally) to generate a KubeConfig based on the KubeadmConfig's information.
+	// The kubeconfig is generated with a server and context matching the KubeadmConfig's name,
```

Comment:
Is this true? If I see correctly we hardcode to "default"

Reply:
Not anymore, removing this